### DISCUSSION TITLE
VoD playlist skips 10 seconds to transition to another video file play

### DISCUSSION THREAD

**Discussion by elijahwere** [ant-media/Ant-Media-Server#6633] [FAQs]
Hello there!
I configured Ant Media for a client to stream VOD playlists to YouTube. But we've noticed that the playlist skips 10 seconds of each video while transitioning to play.

I would appreciate any input on this issue.

Thanks

**Comment by filip-ams** [comment]
Greetings @elijahwere,

I hope you are doing well.

Can you share a bit more details about your current setup, and also the Ant Media server logs at the time of the skips. Providing the VoD files you are trying to stream would also help me reproduce this issue. Thank you in advance.

Best regards,
Filip
